instrumentation/grape/lib/opentelemetry/instrumentation/grape/instrumentation.rb

```diff
@@ -21,18 +22,10 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         end
 
         compatible do
-          # ActiveSupport::Notifications were introduced in Grape 0.13.0
-          # https://github.com/ruby-grape/grape/blob/master/CHANGELOG.md#0130-2015810
           gem_version >= MINIMUM_VERSION
         end
 
-        # Config options available in ddog Grape instrumentation
-        option :enabled, default: true, validate: :boolean
-        option :error_statuses, default: [], validate: :array
-        # Config options necessary for OpenTelemetry::Instrumentation::ActiveSupport.subscribe called in railtie
-        # instrumentation/active_support/lib/opentelemetry/instrumentation/active_support/span_subscriber.rb#L23
-        # Used to validate if any of the payload keys are invalid
-        option :disallowed_notification_payload_keys, default: [], validate: :array
```

Comment:
I decided to leave this one out due to the following reasoning:

IMO, it makes sense that this is used in the ActiveSupport instrumentation because what's going in there is unknown (i.e. could be any notification instrumented by any library or service). However, since in this case we're instrumenting Grape, what Grape is included in the payload is pretty limited and predefined. Because of this, I don't see any risks here.

However, if anybody still sees the value of adding this and can give me an example of a potential case where we might need this, I'd be happy to add it 😄

Reply:
this could be useful if we were adding PII information to the span but we are not so I agree we don't need this key.